VTK does not compile

```
-- CURRENT_INSTALLED_DIR=C:/Libs/vcpkg/installed/x64-windows
-- DOWNLOADS=C:/Libs/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=C:/Libs/vcpkg/packages/vtk_x64-windows
-- CURRENT_BUILDTREES_DIR=C:/Libs/vcpkg/buildtrees/vtk
-- CURRENT_PORT_DIR=C:/Libs/vcpkg/ports/vtk/.
-- Downloading http://www.vtk.org/files/release/7.1/VTK-7.1.0.tar.gz...
-- Downloading http://www.vtk.org/files/release/7.1/VTK-7.1.0.tar.gz... OK
-- Testing integrity of downloaded file...
-- Testing integrity of downloaded file... OK
-- Extracting source C:/Libs/vcpkg/downloads/VTK-7.1.0.tar.gz
-- Extracting done
-- Applying patch C:/Libs/vcpkg/ports/vtk/transfer-3rd-party-module-definitions.patch
-- Applying patch C:/Libs/vcpkg/ports/vtk/transfer-3rd-party-module-definitions.patch done
-- Applying patch C:/Libs/vcpkg/ports/vtk/transfer-hdf5-definitions.patch
-- Applying patch C:/Libs/vcpkg/ports/vtk/transfer-hdf5-definitions.patch done
-- Applying patch C:/Libs/vcpkg/ports/vtk/netcdf-use-hdf5-definitions.patch
-- Applying patch C:/Libs/vcpkg/ports/vtk/netcdf-use-hdf5-definitions.patch done

-- Applying patch C:/Libs/vcpkg/ports/vtk/dont-define-ssize_t.patch
-- Applying patch C:/Libs/vcpkg/ports/vtk/dont-define-ssize_t.patch done
-- Configuring x64-windows-rel
-- Configuring x64-windows-rel done
-- Configuring x64-windows-dbg
-- Configuring x64-windows-dbg done
-- Package x64-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:14 (message):
    Command failed: C:/Program Files/CMake/bin/cmake.exe;--build;.;--config;Release;--target;install;--;-v
    Working Directory: C:/Libs/vcpkg/buildtrees/vtk/x64-windows-rel
    See logs for more information:
      C:\Libs\vcpkg\buildtrees\vtk\package-x64-windows-rel-out.log
      C:\Libs\vcpkg\buildtrees\vtk\package-x64-windows-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:27 (vcpkg_execute_required_process)
  ports/vtk/portfile.cmake:66 (vcpkg_install_cmake)
  scripts/ports.cmake:73 (include)


Error: Building package vtk:x64-windows failed with: BUILD_FAILED
```
*-rel-out.log contains: `LINK : fatal error LNK1181: cannot open input file 'hdf5-shared.lib'`. Indeed, file `hdf5-shared.lib` does not exist on my computer, but rather `hdf5.lib`.

@RobertSpir Did you test VTK installation with shared libraries or only static libraries?

--- a/ports/vtk/CONTROL
+++ b/ports/vtk/CONTROL
@@ -1,4 +1,4 @@
 Source: vtk
-Version: 7.1.0
+Version: 7.1.1
 Description: Software system for 3D computer graphics, image processing, and visualization
 Build-Depends: zlib, libpng, tiff, libxml2, jsoncpp, glew, freetype, expat, hdf5, qt5, msmpi

--- a/ports/vtk/fix-findhdf5-shared.patch
+++ b/ports/vtk/fix-findhdf5-shared.patch
@@ -1,0 +1,20 @@
+diff --git a/CMake/FindHDF5.cmake b/CMake/FindHDF5.cmake
+index 6d558e39b1..9511e9a1ad 100644
+--- a/CMake/FindHDF5.cmake
++++ b/CMake/FindHDF5.cmake
+@@ -3,11 +3,11 @@
+ # (BUG #0014363).
+ 
+ # include the default FindHDF5.cmake.
+-if(CMAKE_VERSION VERSION_LESS 3.6.1)
++#if(CMAKE_VERSION VERSION_LESS 3.6.1)
+   include(${CMAKE_CURRENT_LIST_DIR}/NewCMake/FindHDF5.cmake)
+-else()
+-  include(${CMAKE_ROOT}/Modules/FindHDF5.cmake)
+-endif()
++#else()
++#  include(${CMAKE_ROOT}/Modules/FindHDF5.cmake)
++#endif()
+ 
+ if(HDF5_FOUND AND (HDF5_IS_PARALLEL OR HDF5_ENABLE_PARALLEL))
+   include(vtkMPI)

--- a/ports/vtk/portfile.cmake
+++ b/ports/vtk/portfile.cmake
@@ -1,9 +1,9 @@
 include(vcpkg_common_functions)
-set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/VTK-7.1.0)
+set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/VTK-7.1.1)
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://www.vtk.org/files/release/7.1/VTK-7.1.0.tar.gz"
-    FILENAME "VTK-7.1.0.tar.gz"
-    SHA512 83f0c9291cd2bc517a4bc449ad28a1d77e7ece169e846c7beab0b1140b9ef4b7bad30e3fa6e5a59826a707df78e2e336e07fa8bd30cf7498096ee80feef16534
+    URLS "http://www.vtk.org/files/release/7.1/VTK-7.1.1.tar.gz"
+    FILENAME "VTK-7.1.1.tar.gz"
+    SHA512 34a068801fe45f98325e5334d2569fc9b15ed38620386f1b5b860c9735e5fb8510953b50a3340d3ef9795e22fecf798c25bf750215b2ff1ff1eb7a1ecd87b623
 )
 vcpkg_extract_source_archive(${ARCHIVE})
 
@@ -14,6 +14,7 @@ vcpkg_apply_patches(
         ${CMAKE_CURRENT_LIST_DIR}/transfer-hdf5-definitions.patch
         ${CMAKE_CURRENT_LIST_DIR}/netcdf-use-hdf5-definitions.patch
         ${CMAKE_CURRENT_LIST_DIR}/dont-define-ssize_t.patch
+        ${CMAKE_CURRENT_LIST_DIR}/fix-findhdf5-shared.patch
 )
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)

